bindata/assets/deployments/console-deployment.yaml

```
@@ -55,6 +55,7 @@ spec:
                   - "25"
           name: console
           securityContext:
+            readOnlyRootFilesystem: true
```

we will probably need to switch this to false due to the fact that dev console part is writing into the filesystem
```suggestion
            readOnlyRootFilesystem: false
```